McJon/puplets
# Product reviews

### Goal
Add product review app to store

### Deployment Instructions
- Pull master
- Ensure that your production theme id is correct in `config.yml`
- `./theme deploy --env production`

### Testing Instructions
- [x]  Go the [PDP](https://st4rwz91440h2utm-29911810107.shopifypreview.com/products/blue-silk-tuxedo)
    - Scroll down and write a review
    - Refresh the page and ensure that the review is showing
    - Scroll to the top of the page, ensure that the reviews badge is showing near the price

- [x]  Go to the [Collection page](https://st4rwz91440h2utm-29911810107.shopifypreview.com/collections/all)
   - Ensure that the badges are showing for each product that has a review
   - Ensure that products with no reviews have a review badge

### Tested On
Chrome on mac

### Code Request Quality Checklist


### Reviewers
@sarahvr92
